dotcom-rendering/src/layouts/TagFrontLayout.tsx

```diff
@@ -0,0 +1,328 @@
+import { css } from '@emotion/react';
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import {
+	brandBackground,
+	brandBorder,
+	brandLine,
+	neutral,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { AdSlot } from '../components/AdSlot';
+import { DecideContainerByTrails } from '../components/DecideContainerByTrails';
+import { Footer } from '../components/Footer';
+import { FrontSection } from '../components/FrontSection';
+import { Header } from '../components/Header';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { Island } from '../components/Island';
+import { Nav } from '../components/Nav/Nav';
+import { Section } from '../components/Section';
+import { SubNav } from '../components/SubNav.importable';
+import { TagFrontHeader } from '../components/TagFrontHeader';
+import { TrendingTopics } from '../components/TrendingTopics';
+import { canRenderAds } from '../lib/canRenderAds';
+import { decidePalette } from '../lib/decidePalette';
+import { getEditionFromId } from '../lib/edition';
+import type { NavType } from '../model/extract-nav';
+import type { DCRTagFrontType } from '../types/tagFront';
+import { Stuck } from './lib/stickiness';
+
+interface Props {
+	tagFront: DCRTagFrontType;
+	NAV: NavType;
+}
+
+// const spaces = / /g;
+/** TODO: Confirm with is a valid way to generate component IDs. */
+// const ophanComponentId = (name: string) =>
+// 	name.toLowerCase().replace(spaces, '-');
+
+// const decideAdSlot = (
+// 	index: number,
+// 	isNetworkFront: boolean | undefined,
+// 	collectionCount: number,
+// 	isPaidContent: boolean | undefined,
+// 	format: ArticleDisplay,
+// 	mobileAdPositions: (number | undefined)[],
+// ) => {
+// 	const minContainers = isPaidContent ? 1 : 2;
+// 	if (
+// 		collectionCount > minContainers &&
+// 		index === getMerchHighPosition(collectionCount, isNetworkFront)
+// 	) {
+// 		return (
+// 			<AdSlot
+// 				data-print-layout="hide"
+// 				position="merchandising-high"
+// 				display={format}
+// 			/>
+// 		);
+// 	} else if (mobileAdPositions.includes(index)) {
+// 		return (
+// 			<Hide from="tablet">
+// 				<AdSlot
+// 					index={index}
+// 					data-print-layout="hide"
+// 					position="mobile-front"
+// 					display={format}
+// 				/>
+// 			</Hide>
+// 		);
+// 	}
+// 	return null;
+// };
```

Comment:
I've left this code commented out purposefully - I think at least some of it will still be relevant once we move onto ads but that's not covered in this PR

Reply:
Decided just to remove it : all the logic can be copied back across from front layout anyway